.prettierrc.js

```diff
@@ -1,8 +1,8 @@
-{
```

Comment:
non-blocker: what's the difference between keeping the files as `.prettierrc` instead to change to `.prettierrc.js`

Reply:
I think JSON format is easier to break and make mistakes. Using `.js` we do have linter support and other stuff.